docs/data/date-pickers/time-picker/TimePickerViews.js

```diff
@@ -11,17 +10,11 @@ export default function TimePickerViews() {
       <DemoContainer>
         <NextTimePicker
           label={'"hours", "minutes" and "seconds"'}
-          defaultValue={dayjs('2022-04-07T15:30:25')}
           views={['hours', 'minutes', 'seconds']}
         />
-        <NextTimePicker
-          label={'"hours"'}
-          defaultValue={dayjs('2022-04-07T15:30:25')}
-          views={['hours']}
-        />
+        <NextTimePicker label={'"hours"'} views={['hours']} />
```

Comment:
Maybe this demo should also have `Mobile` picker versions to correctly show what the example is referring to?